plugins/services/src/js/utils/TaskTableUtil.js

```diff
@@ -20,6 +21,14 @@ const TaskTableUtil = {
         return getUpdatedTimestamp(item) || 0;
       }
 
+      if (prop === "zone") {
```

Comment:
🤔 I'm curious why we don't have a case for "region" also?

Reply:
Good catch

Reply:
blocking this PR because of andrews catch.